internal/types/LLMInferenceService.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	"github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
+	"knative.dev/pkg/kmeta"
+)
+
+type LLMInferenceService struct {
+	*v1alpha1.LLMInferenceService
+}
+
+func (llmisvc *LLMInferenceService) GetMaaSRoleName() string {
+	return kmeta.ChildName(llmisvc.Name, "-model-post-access")
+}
+
+func (llmisvc *LLMInferenceService) GetMaaSRoleBindingName() string {
+	return kmeta.ChildName(llmisvc.Name, "-tier-binding")
+}
```

Comment:
Maybe a simple function that takes llmisvc *LLMInferenceService and returns `RoleName` instead is good enough for this use case?